rhba-dev-env/ansible/roles/build_images/templates/build-overrides.j2

```diff
@@ -0,0 +1,882 @@
+#!/usr/bin/env bash
```

Comment:
Why are we duplicating https://github.com/jboss-container-images/jboss-kie-modules/blob/master/tools/build-overrides/build-overrides.sh ? Eventually it will move to kie-cloud-tools, however we shouldn't be copying it here, because then we'll have two different versions to maintain. Can't this script just be pulled in from the existing location?